webapp/components/product-info.js

```diff
@@ -9,6 +9,7 @@ const DisplayNames = (() => {
   ['edge', 'edge-experimental'].forEach(n => m.set(n, 'Edge'));
   ['firefox', 'firefox-experimental'].forEach(n => m.set(n, 'Firefox'));
   ['safari', 'safari-experimental'].forEach(n => m.set(n, 'Safari'));
+  m.set('chrome_android', 'ChromeAndroid');
```

Comment:
Put this after `m.set('android_webview', 'WebView');`

Reply:
sure, will update